README.md

```diff
@@ -45,19 +45,41 @@ Note: Every time you run `git commit` in this repo, it will run some lint checks
 
 Once you have the requirements installed, you should be able to develop by just activating the environment (step 2).
 
-#### Python - adding a new requirement
+### Python - adding a new requirement
 
 1. Update the file `requirements.in` with the name of the library that you want to add.
 1. Install pip-compile if you don't have it already - `pip install pip-tools`.
 1. Run `pip-compile requirements.in`
 1. Download the updated packages `pip install -r requirements.txt`
 1. Check locally to ensure that you haven't broken anything
 
-To run the backend server:
+### Running the backend repo
 
-```
-todo
-```
+#### PyCharm
+
+1. The run configurations should automatically be loaded and working, select the `dev` environment and hit the play button. Contact Jay if you have trouble setting this up.
+
+#### Mac
+
+1. Ensure you are in the root directory, and your virtual env is activated. Also make sure your Postgres is running.
+1. Run `export PYTHONPATH=.`
+1. Run `export CONFIG_PATH=configs/dev-config.yaml`
+1. If you have not done so recently, run `make install` as it will update any schema changes.
+1. Run `python app/__main__.py`
+1. You should see a `SERVER STARTED` message along with configuration details
+
+#### Windows
+
+1. Ensure you are in the root directory, and your virtual env is activated. Also make sure your Postgres is running.
+1. Run `set PYTHONPATH=.`
```

Comment:
If they using bash on windows, it will be `export PYTHONPATH=.` and `export CONFIG_PATH=configs/dev-config.yaml` too, but looks good

Reply:
Does `set` still work if they are using bash on Windows? Or does it have to be `export` strictly?

Reply:
It depends on the terminal they use. For `git bash` only `export` works. For `cmd` i think it is `set`

Reply:
Updated instructions; Windows why are you like this 😞